.github/workflows/website.yml

```diff
@@ -33,25 +33,39 @@ jobs:
         run: |
           pip install -r requirements.txt
 
-      # Action to download artifacts from a different workflow (analysis.yml)
+      # Action Figures artifact
       - name: 'Download artifact'
         if: ${{ github.event.workflow_run.conclusion == 'success' }}
         uses: ./.github/actions/download-artifact
         with:
           name: 'Figures'
+      # Action analysis data artifact
+      - name: 'Download analysis data'
+        if: ${{ github.event.workflow_run.conclusion == 'success' }}
+        uses: ./.github/actions/download-artifact
+        with:
+          name: 'Data'
 
-      - name: Build html
+      - name: move data to the website folder
+        run: |
+          mv test_output.csv website/
+
+      - name: Build documentation
```

Comment:
Maybe I'm not understanding the new folder structure here but it seems like the documentation is getting lost because there's not link on the home page. Maybe there should be a new home that links to everything (dashboard, old figures, documentation)?

Reply:
What do you think of this one? a bit simple and empty but we can fill it with some content from the README
<img width="436" alt="image" src="https://github.com/OSIPI/TF2.4_IVIM-MRI_CodeCollection/assets/62952819/f64469ae-aea0-4178-91e9-e6ad8f0b4274">

Reply:
That would work. Maybe less text - it says OSIPI Taskforce 2.4 3 times in that page. That could probably just be in the title. Is this active on your site? It looks like it's still the previous version.

Reply:
Done. It was not merged yet there but now it is active: https://ahmedbasem20.github.io/TF2.4_IVIM-MRI_CodeCollection/